server/store/user_store.go

```diff
@@ -38,8 +38,9 @@ type User struct {
 	Remote            *remote.User
 	MattermostUserID  string
 	OAuth2Token       *oauth2.Token
-	Settings          Settings          `json:"mattermostSettings,omitempty"`
-	ActiveEvents      []string          `json:"events"`
+	Settings          Settings `json:"mattermostSettings,omitempty"`
+	ActiveEvents      []string `json:"events"`
+	LastStatus        string
```

Comment:
Do you see any value in storing this as a `*model.Status` instead? Currently we don't benefit from it based on how the code is written but I'm just wondering.

Reply:
The only "useful" information for us in model.Status as far as I can see is the Manual bool, but we don't need it after we have decided that it was a manual status. That being said, we could consider storing that if we would like to do some more complex handling in the future (store the status and whether it was manual or not) but I think it is not worth it.